assets/src/edit-story/output/page.js

```diff
@@ -57,6 +48,18 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
         ...generatePatternStyles(backgroundColor),
       };
 
+  const animationDuration = getTotalDuration({ animations }) / 1000;
+  // If the page doesn't have media, take either the animations time or the configured default duration time.
+  const nonMediaPageDuration = Math.max(
```

Comment:
@swissspidy I'm leaving the animation vs default duration logic the same, assuming the default page duration still has priority over the animations. Thoughts?

Reply:
I'll defer to @o-fernandez on this one.

The logic as per Morten's table below is a bit weird currently.